L/LZFSE/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "LZFSE"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/lzfse/lzfse.git", "88e2d2788b4021d0b2eb9fe2d97352ae9190f128")
+]
+
+script = raw"""
+    cd $WORKSPACE/srcdir/lzfse
+
+    if [[ "${target}" == *"freebsd"* ]]; then
+        # Add the flag for FreeBSD targets
+        export CFLAGS="${CFLAGS} -D_XOPEN_SOURCE=700"
+    fi
+
+    # Build with CMake
+    cmake -B build \
+        -DCMAKE_INSTALL_PREFIX=${prefix} \
+        -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_C_FLAGS="${CFLAGS}"
```

Comment:
Technically you don't need this, [`CMAKE_C_FLAGS`](https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_FLAGS.html) is already initialised based on the value of `CFLAGS`.

Reply:
Removed `CMAKE_C_FLAGS` from the list of options